util/chplenv/chpl_gpu.py

```diff
@@ -285,6 +286,8 @@ def validate(chplLocaleModel, chplComm):
     gpu.validate_sdk_version()
 
     if get() == 'cpu':
+        if chpl_tasks.get() == 'fifo':
+            print("WARNING: The 'fifo' tasking model is not supported on with CPU as device mode")
```

Comment:
Not sure if you discussed this with Engin, but I think the warning can be an error, since the behavior is, as far as I know, blatantly incorrect under `fifo`.

Reply:
I can change that to say `ERROR:`. 

Additionally: I'm wondering if even that is enough? I'm not sure if just `printchplenv` printing an extra line is enough for behavior that's blatantly incorrect? Maybe the compiler should also error out?

Reply:
I meant actually exiting with an error from the script :)
Tthere are other cases that do this -- see here, for example: https://github.com/chapel-lang/chapel/blob/c942be6fa36c8f96765e4d1297da43941ea11d99/util/chplenv/chpl_gpu.py#L78-L80

If printchplenv exits with a bad code, the compiler won't run. That's what I meant.

Reply:
Oh I see, I'll do that 👍